instrumentation/aws-lambda/aws-lambda-events-2.2/library/build.gradle.kts

```diff
@@ -18,6 +18,8 @@ dependencies {
   // in public API.
   library("com.amazonaws:aws-lambda-java-events:2.2.1")
 
+  compileOnly("com.amazonaws:aws-lambda-java-serialization:1.1.5")
```

Comment:
Maybe add a comment here about this library being available during runtime natively in lambda?

Reply:
@rapphil done